crates/sui-indexer-alt-framework/src/lib.rs

```diff
@@ -375,7 +388,7 @@ impl Indexer {
         );
 
         if let Some(enabled_pipelines) = &mut self.enabled_pipelines {
-            if !enabled_pipelines.remove(P::NAME) {
+            if P::NAME != CpMapping::NAME && !enabled_pipelines.remove(P::NAME) {
```

Comment:
With this we will always run CpMapping pipeline regardless of enabled pipeline config. Since we may have multiple instances (running pipelines for different tables) writing to the same db, does that mean the cp mapping table will be written to by every instance? This might be fine but I wonder if there's a better way.

Reply:
That's a good call out, I haven't given it too much thought. There seems to be some tradeoff between these indexers being independent, but avoiding redundancy. I figured it'd be simpler for now to just have the framework spin this up. The writes to db do nothing if there are any conflicts

Reply:
That would be quite a waste in resources, since we may have a dozen of the pods running.

Reply:
Removed this, it is now up to the operator to start this pipeline. Pruners will no-op if it cannot find a mapping from the cp_mapping table.